docs/source/config/server.rst

```diff
@@ -0,0 +1,214 @@
+
+``CERT``
+~~~~~~~~
+
+Used to enable SSL and set the type of host certificate.
+The following values are supported:
+
++--------+------------------------------------------------+
+| NONE   | SSL disabled                                   |
++--------+------------------------------------------------+
+| ADHOC  | SSL enabled, autogenerated certificate is used |
++--------+------------------------------------------------+
+| SIGNED | SSL enabled, signed certificate is used        |
++--------+------------------------------------------------+
+
+Default value is ``NONE``.
+
+``CERT_PEM``
+~~~~~~~~~~~~
+
+Path of the certificate file to use for SSL connections.
+
+``CERT_KEY``
+~~~~~~~~~~~~
+
+Path of the private key file to use for SSL connections.
+
+mxcube
+------
+
+The mxcube section allows you to customize miscellaneous aspects of the MxCube application.
+It have settings for the on-axis video stream, user account management, experiment mode.
+The section has the following syntax:
+
+.. code-block:: yaml
+
+    mxcube:
+      mode: <experiment mode>
+      USE_EXTERNAL_STREAMER: <boolean>
+      VIDEO_FORMAT: <format name>
+      VIDEO_STREAM_URL: <stream URL>
+      VIDEO_STREAM_PORT: <stream port>
+      usermanager:
+        class: <class name>
+        inhouse_is_staff: <boolean>
+        users:
+          - username: <user name>
+            role: <role name>
+          ...
+
+The ``mxcube`` section may contain keys as described below.
+
+``mode``
+~~~~~~~~
+
+Set the experiment mode for the application.
+Different modes tailor the presented UI to be suitable for a specific type of MX experiments.
+The following values are supported:
+
++--------------+--------------------------------------+
+| mode         | experiment type                      |
++==============+======================================+
+| OSC          | standard oscillation data collection |
++--------------+--------------------------------------+
+| SSX-CHIP     | SSX fixed target experiments         |
++--------------+--------------------------------------+
+| SSX-INJECTOR | SSX injector experiments             |
++--------------+--------------------------------------+
+
+Default mode is ``OSC``.
+
+``USE_EXTERNAL_STREAMER``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use an external process to stream the sample view video.
+When enabled, MxCube will invoke an appropriate hook on the camera hardware object to start the streamer.
+It is assumed that the camera object supports the external streamer feature.
+
+Default value is ``False``.
+
+``VIDEO_FORMAT``
+~~~~~~~~~~~~~~~~
+
+The format to use when streaming the sample view video.
+The ``MPEG1`` and ``MJPEG`` formats are supported.
+Note that your camera hardware object must support the specified format.
+
+Default format is ``MPEG1``.
+
+``VIDEO_STREAM_URL``
+~~~~~~~~~~~~~~~~~~~~
+
+When set, it will override the default video stream URL.
+The URL that is used by the UI to fetch the sample view video.
```

Comment:
I would swap the two sentences:

```suggestion
The URL that is used by the UI to fetch the sample view video.
When set, it will override the default video stream URL.
```